docs/index.md

```diff
@@ -48,7 +48,6 @@ design goals:
  - **Well-documented**, so that if you *do* go to the documentation, you don't have to search for long.
  - **Extensible**, so you can simply add your own rules, and error messages.
  - **Well-tested**, so that you're absolutely sure that you can rely on the validation rules.
- - **Zero** dependencies, so that you can use it in *any* PHP project.
```

Comment:
Instead of removing it, could you replace this for "Very few dependencies"? It's a nice non functional requirement that I wouldn't want remove completely. You're very right that this should be fixed though! No one likes these kind of surprises :)